LICENSE

```diff
@@ -8,14 +8,14 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor:             Babylonchain, Inc.
+Licensor:             Babylon Labs, Ltd.
 
 Licensed Work:        staking-indexer
-                      The Licensed Work is (c) 2024 Babylonchain, Inc.
+                      The Licensed Work is (c) 2024 Babylon Labs, Ltd.
```

Comment:
just note, we are still waiting from confirmation from Fisher whether it should be 2023 or 2024

Reply:
Thanks for the reminder. Let's wait then

Reply:
merging for now, will change LICENSE after decision is made